.github/workflows/ci.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         run: dotnet build --configuration Release --no-restore -bl:logs/build.binlog
 
       - name: Test
+        uses: ilammy/msvc-dev-cmd@v1  # Enable MSVC on Windows for test suite.
```

Comment:
Pretty sure all this does is run `vcvarsall.bat`. It assumes VS is installed on the machine, which it won't be in GH Actions.

Reply:
VS is available on the Windows agent. Prefer to keep this script in place as it no-ops on Linux automatically.